server/controllers/find/courseOfferingsController.ts

```diff
@@ -13,9 +16,17 @@ export default class CourseOfferingsController {
     return async (req: Request, res: Response) => {
       TypeUtils.assertHasUser(req)
 
-      const course = await this.courseService.getCourseByOffering(req.user.token, req.params.courseOfferingId)
-      const courseOffering = await this.courseService.getOffering(req.user.token, req.params.courseOfferingId)
-      const organisation = await this.organisationService.getOrganisation(req.user.token, courseOffering.organisationId)
+      const [course, [courseOffering, organisation]] = await Promise.all([
+        this.courseService.getCourseByOffering(req.user.token, req.params.courseOfferingId),
+        this.courseService.getOffering(req.user.token, req.params.courseOfferingId).then(async myCourseOffering => {
+          const myOrganisation = await this.organisationService.getOrganisation(
+            req.user.token,
+            myCourseOffering.organisationId,
+          )
+          return [myCourseOffering, myOrganisation] as orgCourseOfferings
```

Comment:
Is there a better affix here than `my` for these inner variables? `unreturned`? `withinPromise` (as a suffix)? It's a tricky one!